evm_loader/cli/src/main.rs

```diff
@@ -416,7 +416,7 @@ fn make_deploy_ethereum_transaction(
         let tx = UnsignedTransaction {
             to: None,
             nonce: trx_count,
-            gas_limit: 1.into(),
+            gas_limit: 9_999_999.into(),
```

Comment:
@s-medvedev Shouldn't the gas limit be a configurable in the CLI?

Reply:
Yes, we will need the ability to set it through an argument when calling the contract deployment. But this should be done as a separate task. Here is another related task - how can the user estimate the required amount of gas?